dowhy/causal_refuters/bootstrap_refuter.py

```diff
@@ -128,8 +128,12 @@ def _refute_once(
                 new_data[variable] = np.where(probs < probability_of_change, changed_data)
                 new_data[variable].astype("category")
 
-    new_estimator = CausalEstimator.get_estimator_object(new_data, target_estimand, estimate)
-    new_effect = new_estimator.estimate_effect()
+    new_estimator = estimate.estimator.get_estimator_object(new_data, target_estimand, estimate)
```

Comment:
I think we should have an explicit fit method here, to be consistent with the new API.

Reply:
and then make sure that the right effect modifiers and other parameters are passed to the fit method.